src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -0,0 +1,115 @@
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Image;
+
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\Util\StringUtil;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * This form type integrates a Dropzone used to manage images, the twig templates is a simple div
+ * with data attributes, it needs javascript from the Dropzone.vue component to work.
+ *
+ * This form type though should be used to pass any configuration to the javascript components, like
+ * translations, locales, or productId (maybe even urls could be configurable).
+ */
+class ImageDropzoneType extends TranslatorAwareType
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefaults([
+            'translations' => [
+                'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
+                'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
+                'window.selectedFiles' => $this->trans(
+                    '[1]%filesNb%[/1] selected file(s)',
+                    'Admin.Actions',
+                    ['[1]' => '<span>', '[/1]' => '</span>']
+                ),
+                'window.useAsCover' => $this->trans('Use as cover image', 'Admin.Actions'),
+                'window.saveImage' => $this->trans('Save image settings', 'Admin.Actions'),
+                'window.delete' => $this->trans('Delete selection', 'Admin.Actions'),
+                'window.close' => $this->trans('Close window', 'Admin.Actions'),
+                'window.zoom' => $this->trans('Zoom on selection', 'Admin.Actions'),
+                'modal.close' => $this->trans('Close', 'Admin.Actions'),
+                'modal.accept' => $this->trans('Accept', 'Admin.Actions'),
+                'modal.message' => $this->trans('Test of message', 'Admin.Actions'),
+                'modal.title' => $this->trans('Are you sure about deleting %filesNb% file(s)?', 'Admin.Actions'),
```

Comment:
@Julievrz @MatShir I added modal wording there, could you please help me for these wordings?

Reply:
Here's how it looks so you understand the context:
![Capture d’écran 2021-03-24 à 20 19 29](https://user-images.githubusercontent.com/13801017/112370985-587ac000-8cde-11eb-8e61-5b873125e23c.png)

Reply:
I'd say only title could be enough, I wonder if we should change the class of the submit button?